### ISSUE TITLE
[bug]GetState is may crash in concurrency program

### ISSUE THREAD

**Issue by dannypsnl** [dannypsnl/redux#27]
## Problem
As title
#### How error happen
If you write down the code like
```go
for i:=0; i<10; i++ {
    go func() {
        store.Dispatch(action.New("INC"))
    }()
}
store.GetState("counter")
```
The `GetState` is obviously could touch map with `go func` at the same time.
It will cause runtime panic. So use `sync.Map` is must to do.

**Comment by dannypsnl** [comment]
Fix in v1.2.0

**Comment by dannypsnl** [comment]
But it won't happen XD. Because Dispatch has the lock.
